Modules/clinic/posixmodule.c.h

```diff
@@ -3705,6 +3705,50 @@ os_pread(PyObject *module, PyObject *const *args, Py_ssize_t nargs)
 
 #endif /* defined(HAVE_PREAD) */
 
+#if defined(HAVE_PREADV2)
```

Comment:
You use `HAVE_PREADV2`, but I don't see anything to actually set it. Did you forget to update `configure.ac`?